includes/Core/REST_API/REST_Routes.php

```diff
@@ -343,6 +343,22 @@ private function get_routes() {
 					),
 				)
 			),
+			new REST_Route(
+				'modules',
+				array(
+					array(
+						'methods'             => WP_REST_Server::READABLE,
+						'callback'            => function( WP_REST_Request $request ) {
+							$modules = $this->modules->get_available_modules();
+							return new WP_REST_Response( array_values( array_map( array( $this, 'prepare_module_data_for_response' ), $modules ) ) );
+						},
+						'permission_callback' => $can_authenticate,
+					),
+				),
+				array(
+					'schema' => $this->get_module_schema(),
```

Comment:
Should this be the same for returning a collection of modules as the schema used for returning a single module?

Reply:
Per the way WP core does it, they provide the same schema for both collections and singles.